### ISSUE TITLE
Score of solutions

### ISSUE THREAD

**Issue by thiagosalper** [rails-projects-br/decide#5]
We have to create a score system to solutions/questions. This system needs to contain 'user_id' and 'solution_id', to not allow a repeated score per user.

**Comment by filabreu** [comment]
Great! Will describe as User Stories.

A User should be able to score up a Problem.
A User should be able to score up a Solution.
A User can score up a same Problem only once.
A User can score up a same Solution only once.
A User can score up only one Solution per Problem (?).

What do you think about the last bussiness logic I described?

**Comment by RodrigoPatrao** [comment]
About the last BL: if an user vote for a solution, and after his vote a better solution appears and he wants to change its vote, how this situation will be handled?

**Comment by filabreu** [comment]
Yes, I believe so. Because think about this: we should implement the solution. It is already hard to do one, expect more than one. Specially if we are counting on collaborative/volunteering work like ours. I guess that this would provide a difference from StackOverflow and similiar platforms.

But this is just my opinion, we could go on different paths here.
